src/traces/indicator/attributes.js

```diff
@@ -0,0 +1,408 @@
+            'in this case, it is always centered'
+        ].join(' ')
+    },
+    // position
+    domain: domainAttrs({name: 'indicator', trace: true, editType: 'calc'}),
+
+    vmin: {
+        valType: 'number',
+        editType: 'calc',
+        role: 'info',
+        dflt: 0,
+        description: [
+            'Sets the minimum value of the gauge.'
+        ].join(' ')
+    },
+    vmax: {
+        valType: 'number',
+        editType: 'calc',
+        role: 'info',
+        description: [
+            'Sets the maximum value of the gauge.'
+        ].join(' ')
+    },
+
+    title: {
+        text: {
+            valType: 'string',
+            role: 'info',
+            editType: 'plot',
+            description: [
+                'Sets the title of this indicator.'
+            ].join(' ')
+        },
+        align: {
+            valType: 'enumerated',
+            values: ['left', 'center', 'right'],
+            role: 'info',
+            editType: 'plot',
+            description: [
+                'Sets the horizontal alignment of the title.',
+                'It defaults to `center` except for bullet charts',
+                'for which it defaults to right.'
+            ].join(' ')
+        },
+        font: extendFlat({}, textFontAttrs, {
+            description: [
+                'Set the font used to display the title'
+            ].join(' ')
+        }),
+        editType: 'plot'
+    },
+    number: {
+        valueformat: {
+            valType: 'string',
+            dflt: '.3s',
+            role: 'info',
+            editType: 'plot',
+            description: [
+                'Sets the value formatting rule using d3 formatting mini-language',
+                'which is similar to those of Python. See',
+                'https://github.com/d3/d3-format/blob/master/README.md#locale_format'
+            ].join(' ')
+        },
+        font: extendFlat({}, textFontAttrs, {
+            description: [
+                'Set the font used to display main number'
+            ].join(' ')
+        }),
+        suffix: {
+            valType: 'string',
+            dflt: '',
+            role: 'info',
+            editType: 'plot',
+            description: [
+                'Sets a suffix appearing next to the number.'
+            ].join(' ')
+        },
+        editType: 'plot'
+    },
+    delta: {
+        reference: {
+            valType: 'number',
+            role: 'info',
+            editType: 'calc',
+            description: [
+                'Sets the reference value to compute the delta.'
+            ].join(' ')
+        },
+        position: {
+            valType: 'enumerated',
+            values: ['top', 'bottom', 'left', 'right'],
+            role: 'info',
+            dflt: 'bottom',
+            editType: 'plot',
+            description: [
+                'Sets the position of delta with respect to the number.'
+            ].join(' ')
+        },
+        relative: {
```

Comment:
Just to document something me and @antoinerg discussed in-person last week.

----------------

One potential addition to `indicator` would be to add a way to show both "absolute" and "relative" delta widgets on the trace same `indicator` trace. 

One way to achieve this would be to add `'delta2'` flag in mode and have something like:

```js
{
  mode: 'number+delta+delta2'
  delta: {},
  delta2: {relative: true}
}
```

OR, replace `delta` with `deltarel` and `deltaabs`

```js
{
  mode: 'number+deltarel+deltaabs'
  deltarel: {},
  deltaabs: {}
}
```

making this `relative` attribute obsolete.

Reply:
That's definitely a possibility! 

I think I prefer having `delta2` as it would imply it is to be displayed to the right of `delta`. If we have `deltarel` and `deltaabs`, we'd have to add an attribute to set their position relative to one another. Basically, we'd be trading attribute `relative` for `relativeposition`...